scene/gui/color_picker.cpp

```diff
@@ -39,6 +39,8 @@
 #include "editor_settings.h"
 #endif
 #include "scene/main/viewport.h"
+#include <stdio.h>
+#include <string>
```

Comment:
Godot uses own implementation of string https://github.com/godotengine/godot/blob/master/core/ustring.h
also stdio.h is probably not needed